keyvi/src/cpp/dictionary/fsa/internal/json_value_store.h

```diff
@@ -27,6 +27,8 @@
 
 #include <boost/functional/hash.hpp>
 #include <zlib.h>
+
+#define RAPIDJSON_SSE42
```

Comment:
yes, they have some SSE optimizations, not sure if it will speed up our usecases, but they might optimize other parts later on.